Unable to install on windows, error on 'postinstall'

When I try to install, the following error occurs:

```
$ npm install protractor-cucumber-steps

> protractor-cucumber-steps@1.3.7 postinstall C:\git\request-axios\node_modules\protractor-cucumber-steps
> cd ./node_modules/protractor; npm install webdriver-manager@latest; cd ../.. && node_modules/.bin/webdriver-manager update

O sistema n▒o pode encontrar o caminho especificado.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.2 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! protractor-cucumber-steps@1.3.7 postinstall: `cd ./node_modules/protractor; npm install webdriver-manager@latest; cd ../.. && node_modules/.bin/webdriver-manager update`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the protractor-cucumber-steps@1.3.7 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Paulo Gonçalves\AppData\Roaming\npm-cache\_logs\2020-04-07T18_57_43_525Z-debug.log
```

Changes to make:
```diff
-postinstall": "cd ./node_modules/protractor; npm install webdriver-manager@latest; cd ../.. && node_modules/.bin/webdriver-manager update"
+postinstall": "cd ./node_modules/protractor; npm install webdriver-manager@latest; cd ../..; node_modules/.bin/webdriver-manager update"
```

---
**OS:** Windows 10 Home Language
**npm version:** 6.14.4
**node version:** 12.16.1
**protractor-cucumber-steps version:** 1.3.7

--- a/package.json
+++ b/package.json
@@ -7,7 +7,7 @@
     "node": ">=6.9.x"
   },
   "scripts": {
-    "postinstall": "cd ./node_modules/protractor; npm install webdriver-manager@latest; cd ../.. && node_modules/.bin/webdriver-manager update",
+    "postinstall": "npx webdriver-manager update",
     "lint": "node_modules/.bin/eslint *.js tests/**/*.js",
     "test": "node_modules/.bin/protractor tests/protractor.conf.js",
     "patch": "npm version patch -m \"Bumped up package version to %s\" && git push && git push origin --tags && npm publish",

--- a/package.json
+++ b/package.json
@@ -50,5 +50,8 @@
     "node-testing-server": "^1.3.1",
     "protractor": "^5.4.2",
     "protractor-cucumber-framework": "^6.1.3"
+  },
+  "peerDependencies": {
+    "protractor": ">= 5.0.0"
   }
 }

